src/components/Article/Article.jsx

```diff
@@ -1,6 +1,69 @@
-// import the React library here
-// import a css file containig article styles
+import React from 'react';
 
-// Create an Article function returning the HTML of article block
+import './Article.css';
 
-// Add a default export statement for Article component to use it in the other files
+function Article() {
+  return (
+    <article className="article">
```

Comment:
Usually we use arrow functions for components. Also you can export right away
```suggestion
export const Article = () => (
    <article className="article">
```